receiver/README.md

```diff
@@ -1,11 +1,27 @@
-A variety of receivers are available to the OpenCensus Service (both Agent and Collector)
+**Note** This documentation is still in progress. For any questions, please reach out in the
+[OpenTelemetry Gitter](https://gitter.im/open-telemetry/opentelemetry-service) or
+refer to the [issues page](https://github.com/open-telemetry/opentelemetry-service/issues).
 
-__Currently there are some inconsistencies between Agent and Collector configuration, those will be addressed by issue
-[#135](https://github.com/census-instrumentation/opencensus-service/issues/135).__ 
+# Receivers
+A receiver is how data gets into OpenTelemetry Service. Generally, a receiver accepts data in a specified format and can
+support [traces](https://github.com/open-telemetry/opentelemetry-proto/blob/2ccb7cb0cf038086955c46eebea767fddb331d16/src/opentelemetry/proto/trace/v1/trace.proto) and/or [metrics](https://github.com/open-telemetry/opentelemetry-proto/blob/2ccb7cb0cf038086955c46eebea767fddb331d16/src/opentelemetry/proto/metrics/v1/metrics.proto).
```

Comment:
Unclear why this links to a specific commit of draft OpenTelemetry proto repo. A trace can come from non-OpenTelemetry protocols too.

I would like to generic definition of a trace (if a link is needed).

Reply:
gah - another example of gh convo's being shrunk/moved after a change! 
This is the first set of 'docs' I could find to talk about traces/metrics with the help of the reviewers of the pr. Is there a document that talks about traces/metrics at a higher level and that isn't otel specific?

Reply:
I am not aware of a document outside otel that is useful to link from here. Inside otel perhaps check the specification repo and see if any of the documents there is appropriate for linking.

Reply:
Hmmm - i can't find anything either. There needs to be somewhere that defines what is meant by traces/metrics. I'll remove the hyperlinks for now.